po/de.po

```diff
@@ -8249,213 +8226,232 @@ msgstr ""
 "wollen? \n"
 "Alle persönlichen Änderungen gehen hierdurch verloren."
 
-#: ../src/gui/preferences.c:1731
+#: ../src/gui/preferences.c:1706
 #, c-format
 msgid "failed to import preset %s"
 msgstr "Voreinstellung %s konnte nicht importiert werden"
 
 #. Zero value indicates import
-#: ../src/gui/preferences.c:1742
+#: ../src/gui/preferences.c:1717
 msgid "select preset to import"
 msgstr "Voreinstellung zum importieren wählen"
 
-#: ../src/gui/preferences.c:1761 ../src/libs/styles.c:547
+#: ../src/gui/preferences.c:1736 ../src/libs/styles.c:547
 msgid "darktable style files"
 msgstr "darktable-Stil-Datei"
 
-#: ../src/gui/preferences.c:1766 ../src/iop/lut3d.c:1588
+#: ../src/gui/preferences.c:1741 ../src/iop/lut3d.c:1588
 #: ../src/libs/copy_history.c:157 ../src/libs/geotagging.c:968
 #: ../src/libs/styles.c:552
 msgid "all files"
 msgstr "alle Dateien"
 
-#: ../src/gui/preferences.c:1914 ../src/gui/presets.c:413
+#: ../src/gui/presets.c:58
+msgid "normal images"
+msgstr "normale Bilder"
+
+#: ../src/gui/presets.c:58 ../src/libs/metadata_view.c:445
+msgid "raw"
+msgstr "Raw"
+
+#: ../src/gui/presets.c:58
+msgid "HDR"
+msgstr "HDR"
+
+#: ../src/gui/presets.c:58 ../src/iop/monochrome.c:77 ../src/libs/image.c:602
+#: ../src/libs/metadata_view.c:453
+msgid "monochrome"
+msgstr "Monochrom"
+
+#: ../src/gui/presets.c:161
+#, c-format
+msgid "preset `%s' is write-protected, can't delete!"
+msgstr ""
+"Voreinstellung „%s” ist schreibgeschützt und kann nicht gelöscht werden"
```

Comment:
Using double quotes inside of double quotes? Could this work?

Reply:
this wasn‘t changed - it just changed the place in the po file. 
Since these are different glyphs (&*dquo) its not the same as straight double quotes.

Reply:
> Since these are different glyphs (&*dquo) its not the same as straight double quotes.
My fault. I have not read exactly enought. Sorry for the noise. ;-)